nixos/modules/services/backup/burpServer.nix

```diff
@@ -0,0 +1,83 @@
+{ config, lib, networking, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.burpServer;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.burpServer = {
+
+      enable = mkOption {
+        default = false;
+        description = "Whether to enable burp.";
+      };
+
+      user = mkOption {
+        default = "burp";
+        description = "The user to run burp as";
+      };
+
+      group = mkOption {
+        default = cfg.user;
+        description = "The group to run burp as (default same as user)";
+      };
+
+      configFile = mkOption {
```

Comment:
The server and client configuration file usually contains passwords for the first authentication during which ssl certificates are generated and signed, so it's probably not a good idea to put that into the world-readable nix-store.